### ISSUE TITLE
Non-conventional code formatting in generated config file

### ISSUE THREAD

**Issue by jehoshua02** [garris/BackstopJS#22]
The config file uses tabs instead of spaces. I think that two spaces is more common and widely accepted among front-end developers.

Also, the commas are on the beginning of the next line, when typically I find them on the end of the line.

This is how I would like to format it:

```json
{
  "viewports": [
    {
      "name": "phone",
      "viewport": {
        "width": 320,
        "height": 480
      }
    }, {
      "name": "tablet_v",
      "viewport": {
        "width": 568,
        "height": 1024
      }
    }, {
      "name": "tablet_h",
      "viewport": {
        "width": 1024,
        "height": 768
      }
    }
  ],
  "grabConfigs": [
    {
      "testName": "http://getbootstrap.com",
      "url": "http://getbootstrap.com",
      "hideSelectors": [],
      "removeSelectors": [
        "#carbonads-container"
      ],
      "selectors": [
        "header", "main", "body .bs-docs-featurette:nth-of-type(1)", "body .bs-docs-featurette:nth-of-type(2)", "footer", "body"
      ],
      "readyEvent": null,
      "delay": 500
    }
  ]
}
```

**Comment by lewisnyman** [comment]
Sounds good, the only thing I would change is to move the opening brackets of an object to a new line, eg:

```
    }, 
    {
      "name": "tablet_v",
      "viewport": {
        "width": 568,
        "height": 1024
      }
    }
```

**Comment by jehoshua02** [comment]
@lewisnyman I can live with that! In fact, that's typically what I do when working in json.

**Comment by garris** [comment]
Works for me.   

@lewisnyman Maybe since you put in the last change on the config file (already merged to master) you can make this update to the config and update the readme doc?   I would pull that right away and push a new release.

**Comment by garris** [comment]
Closed by #29 

thanks @jehoshua02 !
